docs/tuning_fledge.rst

```diff
@@ -762,6 +767,113 @@ When collection is enabled the following counters will be collected for the stor
+
++---------------+
+| |PurgeConfig| |
++---------------+
+
+The details of each of the options are covered elsewhere in the documentation, but the salient points will be repeated here. The operation of the purge process reduces the number of readings that are retained in the readings storage subsystem using two parameters;
+
+   - the age of the reading
+
+   - the number of readings
+
+The age is set in hours and any reading older than this age is a candidate to be removed from the readings data. The purge process also looks at the number of readings stored and will remove the oldest, even if they are newer than the age to be retained if the number exceeds the maximum rows to retain value.
+
+These are the candidates to be removed, but may not be removed depending upon the sent status of the readings and the configuration item *Retain Unsent Data*.
+
+Candidate data that has already been sent to all the defined north destinations in the system will always be removed regardless of the *Retain Unsent Data* setting. Data that has not be marked as a candidate for removal will be retained event after it has been sent to all the north destinations.
+
+If the *Retain Unsent Data* setting is set to *retain unsent to any destination*, then candidate data will be removed if it has been sent to at least one north destination.
+
+As with the purge system process the purge process is also run by a schedule that is accessed via the *Schedules* menu item.
+
++------------------+
+| |PurgeSchedules| |
++------------------+
+
+The frequency of running the purge process is very important, since it as the same effect as described for the purge system execution, but the impact is much higher. Consider a system that wants to retain data for 12 hours, if the purge process is set to run every 12 hours the number of readings over time would be as shown in the graph below
+
++---------------+
+| |PurgeCycles| |
++---------------+
+
+The red line indicates the configured retention point for the readings. Each point where the blue line drops is an execution of the purge process.
+
+This assumes we started with a system with no readings, we read in data for 12 hours and then run the purge process. This is shown as removing a small number of readings to reduce the retain readings to those less than 12 hours old. The initial run is in fact not likely to find any data to remove, or at most a handful of readings, depending on how long it takes the purge process to start executing.
+
+The system now continues to ingest data and will accumulate another 12 hours of data before purge is run again and the data reduced to the newest 12 hours of data.
+
+.. note::
+
+   We are assuming that either unsent data is not retained or we are sending all data north immediately it is received.
```

Comment:
sending all data north immediately it is received -> sending all data north immediately as it is received